helm-charts/splunk-otel-collector/templates/config/_common.tpl

```diff
@@ -101,6 +101,49 @@ resource/logs:
     - key: istio_service_name
       action: delete
     {{- end }}
+    {{- if .Values.splunkPlatform.fieldNameConvention.renameFieldsSck }}
+    - key: container_name
+      from_attribute: k8s.container.name
+      action: upsert
+    - key: cluster_name
+      from_attribute: k8s.cluster.name
+      action: upsert
+    - key: container_id
+      from_attribute: k8s.container.id
```

Comment:
This attribute is actually recorded as `container.id`. Please also change it below with `delete` action